### PR TITLE
Fix Multicompany Issue

### PR DESCRIPTION
**Pull request by mapiolca**
Before : Show all order status for all entities and administrators are repeated in allowed users list

Now : Show only oder status for the current entity and Multicompany add DISTINCT selection when activated.

### CODE REVIEW

File: htdocs/fourn/commande/index.php [hunk 1]
```diff
@@ -162,7 +162,8 @@
 $sql.= ", ".MAIN_DB_PREFIX."commande_fournisseur as cf";
 if (!$user->rights->societe->client->voir && !$socid) $sql.= ", ".MAIN_DB_PREFIX."societe_commerciaux as sc";
 $sql.= " WHERE cf.fk_soc = s.rowid";
-$sql.= " AND s.entity = ".$conf->entity;
+$sql.= " AND s.entity IN (".getEntity("supplier_order").")";
```
Comment (hregis) [inline htdocs/fourn/commande/index.php]: @mapiolca 
replace with : 
`$sql.= " AND s.entity IN (".getEntity("societe").")";`

File: htdocs/fourn/commande/index.php [hunk 2]
```diff
@@ -245,13 +246,17 @@
...
+}
+$sql.= " u.rowid, u.lastname, u.firstname, u.email";
 $sql.= " FROM ".MAIN_DB_PREFIX."user as u,";
 $sql.= " ".MAIN_DB_PREFIX."user_rights as ur";
 $sql.= ", ".MAIN_DB_PREFIX."rights_def as rd";
 $sql.= " WHERE u.rowid = ur.fk_user";
-$sql.= " AND (u.entity IN (0,".$conf->entity.")";
-$sql.= " AND rd.entity = ".$conf->entity.")";
+$sql.= " AND (u.entity IN (0,".getEntity("supplier_order").")";
+$sql.= " AND rd.entity = ".getEntity("supplier_order").")";
```
Comment (hregis) [inline htdocs/fourn/commande/index.php]: @mapiolca 
can you try this instead ?

```
$sql.= " AND (u.entity IN (0,".$conf->entity.")";
$sql.= " AND ur.entity = ".$conf->entity.")";
```

**Comment by hregis** [comment]
@mapiolca 
can you try with this fix : #13130

**Comment by eldy** [comment]
Seems replaced with #13130

**Pull request closed without merging**